.github/workflows/govulncheck.yml

```diff
@@ -23,4 +23,4 @@ jobs:
           go install golang.org/x/vuln/cmd/govulncheck@latest
       - name: Run security checks
         run: |
-          govulncheck -v ./...
+          govulncheck ./...
```

Comment:
> The -v flag causes govulncheck to output more information when run on source. It has no effect when run on a binary.

We may lose insight into codepaths that are affected by reducing verbosity.

Reply:
https://github.com/kubernetes/kube-state-metrics/actions/runs/5502704010/jobs/10027175265 it looks like `-v` got removed from recent versions of govulncheck

Reply:
Ah! Makes sense.